data/json/items/resources/metal.json

```
@@ -589,5 +589,455 @@
     "color": "light_gray",
     "ammo_type": "components",
     "qualities": [ [ "HAMMER", 1 ] ]
+  },
+  {
+    "id": "5kgaluminum",
+    "type": "AMMO",
+    "category": "spare_parts",
+    "name": { "str_sp": "5 l aluminum ingot" },
+    "description": "A 5 L ingot of solid aluminum.",
```

```suggestion
    "name": { "str_sp": "5L aluminum ingot" },
    "description": "A five-liter ingot of solid aluminum.",
```
Proper gramma'

---

```suggestion
    "name": { "str": "5L aluminum ingot" },
    "description": "A five-liter ingot of solid aluminum.",
```
Use "str" to generate the correct plural. (And the same for other items.)